src/ui/map.js

```diff
@@ -507,13 +531,17 @@ class Map extends Camera {
     /**
      * Returns the map's geographical bounds. When the bearing or pitch is non-zero, the visible region is not
      * an axis-aligned rectangle, and the result is the smallest bounds that encompasses the visible region.
+     * @example
+     * map.getBounds();
```

Comment:
Perhaps worth adding `var bounds = ...` in examples like this to show that it returns a value?